packages/react/src/hooks/__tests__/useMediaQuery.test.tsx

```diff
@@ -0,0 +1,68 @@
+import { cleanup } from '@testing-library/react';
+import { renderHook } from '@testing-library/react-hooks';
+
+import { useMediaQueryLessThan, useMediaQueryGreaterThan } from '../useMediaQuery';
+
+/* This test suite changes window width but it's set to default 1024 after this is done. */
+describe('useMediaQuery', () => {
+  afterAll(cleanup);
+
+  it('useMediaQueryLessThan returns correct value', () => {
+    // Change the viewport to 500px
+    global.innerWidth = 500;
+    // Trigger the window resize event
+    global.dispatchEvent(new Event('resize'));
+
+    const isSmallView = renderHook(() => useMediaQueryLessThan('m'));
+    const isMobileView = renderHook(() => useMediaQueryLessThan('xs'));
+
+    expect(isSmallView.result.current).toBe(true);
+    expect(isMobileView.result.current).toBe(false);
+  });
+
+  it('useMediaQueryLessThan reacts to window resize', () => {
+    // Set the viewport width again just to be safe
+    global.innerWidth = 600;
+    // Trigger the window resize event
+    global.dispatchEvent(new Event('resize'));
+
+    const isMediumView = renderHook(() => useMediaQueryLessThan('m'));
+    expect(isMediumView.result.current).toBe(true);
+
+    // Set the viewport width again just to be safe
+    global.innerWidth = 1000;
+    // Trigger the window resize event
+    global.dispatchEvent(new Event('resize'));
```

Comment:
Would [fireEvent](https://testing-library.com/docs/dom-testing-library/api-events#fireevent) be a better alternative here?

Reply:
Good find! [Implemented.](https://github.com/City-of-Helsinki/helsinki-design-system/pull/960/commits/1bf41b423e34bdf3f6483d328b72871bcaf941d4)